### DISCUSSION TITLE
Returned modal object after ADDing to DB

### DISCUSSION THREAD

**Discussion by leigh-pointer** [oqtane/oqtane.framework#1019] [General]
I have used the Module Creator and when adding “ ADD “a new item to the dB the returned instance is empty.  I was expecting the instance to return a populated instance with the newly created identifier Id that I have need to create a child record. 

Is this buy design?

**Comment by hishamco** [comment]
If I'm not wrong it should retrieve the updated version

**Comment by leigh-pointer** [comment]
> If I'm not wrong it should retrieve the updated version

That is exactly what I thought.  I will continue with investigation.

**Reply by leigh-pointer** [reply]
Closing everything down and giving my machine some resource the updated instance is returned. so all good now,
